migen/build/platforms/sinara/metlino.py

```diff
@@ -102,6 +102,43 @@
         Subsignal("rxn", Pins("E3"))
     ),
 
+    ("sfp_ctl", 0,
+        Subsignal("mod_def0", Pins("K27")),
+        Subsignal("mod_def1", Pins("G25")),
+        Subsignal("mod_def2", Pins("H26")),
+        Subsignal("los", Pins("K26")),
+        Subsignal("rate_select", Pins("H23")),
+        Subsignal("tx_disable", Pins("H27")),
+        Subsignal("tx_fault", Pins("G27")),
+        Subsignal("led1", Pins("J24")),
```

Comment:
That's inconsistent with Sayma which has the SFP LEDs defined as ``user_led``.
Kasli follows your style on the other hand, so we can keep it and change Sayma.